logistics/scripts/activities/build/initialize-database/initialize-empty-database.ps1

```diff
@@ -7,8 +7,10 @@
 $ErrorActionPreference = 'Stop'
 $Error.Clear()
 
-& "$PSScriptRoot\..\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
-Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'Application\SolutionScripts\InitializeDevelopmentEnvironment.psm1')
+& "$PSScriptRoot/../../../../../logistics/scripts/modules/load-path-resolver.ps1"
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1')
+
+Write-Warning "logistics/scripts/activities/build/initialize-database/initialize-empty-database.ps1 is deprecated, and will be removed in the near future. Please use the 'Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1' instead."
```

Comment:
More a question of style, this warning doesn't wrap the embedded path in the message inside single quotes like other messages.